hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -282,7 +284,8 @@ public static void main(final String... args) throws Exception {
         // --- Initialize the platform metrics and the Hedera instance ---
         setupGlobalMetrics(platformConfig);
         metrics = getMetricsProvider().createPlatformMetrics(selfId);
-        hedera = newHedera(metrics);
+        final PlatformStateFacade platformStateFacade = new PlatformStateFacade(ServicesSoftwareVersion::new);
```

Comment:
Will this `ServicesSoftwareVersion::new` eventually be replaced with the correct version?

Reply:
Discussed offline. Version is intentionally not set until `platformStateFacade#creationSemanticVersionOf`